docs/api/lights/SpotLight.html

```diff
@@ -92,11 +92,11 @@
 			Default — *Math.PI/3*.
 		</div>
 
-		<h3>[property:Float exponent]</h3>
+		<h3>[property:Float penumbra]</h3>
 		<div>
 			Rapidity of the falloff of light from its target direction. A lower value spreads out the light, while a higher
 			focuses it towards the center.<br />

```

Comment:
Percent of the spotlight cone that is attenuated due to penumbra. Takes values between zero and 1.<br />